src/Http/Controllers/Configuration/SsoController.php

```diff
@@ -49,7 +62,26 @@ public function getConfigurationHome()
     public function postUpdateScopes(SsoScopes $request)
     {
 
-        setting(['sso_scopes', $request->input('scopes')], true);
+        $scopes = collect(setting('sso_scopes', true));
+
+        // default profile cannot be renamed
+        $default_profile = $scopes->first(function ($item) {
+            return $item->name == 'default';
+        });
+
+        if($default_profile->id == $request->input('profile_id') && $request->input('profile_name') != 'default')
```

Comment:
You add a `default` property to your object, why not using it instead the name ?
I think it would be more reliant ;)